react/features/base/participants/middleware.js

```diff
@@ -196,6 +196,13 @@ StateListenerRegistry.register(
                 JitsiConferenceEvents.PARTICIPANT_PROPERTY_CHANGED,
                 (participant, propertyName, oldValue, newValue) => {
                     switch (propertyName) {
+                    case 'features_jigasi':
+                        store.dispatch(participantUpdated({
+                            conference,
+                            id: participant.getId(),
+                            isJigasi: true
+                        }));
+                        break;
```

Comment:
What is this for?

Reply:
I see. Probably specifically wanting to know a phone user to show a phone icon.

Reply:
I guess this depends on https://github.com/jitsi/lib-jitsi-meet/pull/928. Isn't it @zbettenbuk ?

Reply:
Here new value can be true or false, not that we have such case right now, it will always be fired with true only. Just mentioning.

Reply:
Thanks for warning, I replaced ```true``` with ```newValue```